src/styles/main.css

```diff
@@ -0,0 +1,126 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Montserrat:wght@100;700&display=swap');
+
+:root {
+  --pale-gray: #f5f6fa;
+  --azure: #0290ff;
+  --white: #e8e8e8;
+  --medium-blue: #307bbe;
+  --dodger-blue: #379cf6;
+}
+
+.Text-Style-5 {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.flex {
+  display: flex;
+}
+
+.panel-bg {
```

Comment:
Your final design should match Zeplin's design.

- [ ] Please make your navbar exactly the same as in the Zeplin (including Capitalized 'CMS', font size, missing icon, etc.);

Here is your navbar:
<img width="1257" alt="Screen Shot 2021-11-06 at 01 13 56" src="https://user-images.githubusercontent.com/60043698/140573268-1a562274-a9f9-40ab-b2c9-da070e963a26.png">

Here is how it shall look:
<img width="961" alt="Screen Shot 2021-11-06 at 01 14 10" src="https://user-images.githubusercontent.com/60043698/140573304-8ddcf5af-c2d3-40ea-8e84-22ce8a6ebb5d.png">

Reply:
Thanks for the review. I have fixed up the navbar design to match Zeplin's